tests/pages/pmmInventoryPage.js

```diff
@@ -233,9 +233,11 @@ module.exports = {
     );
 
     /* we are using count 7 because we have two agents for RDS Instance also,
-    hence (pmm-agent, Node exporter, postgres exporter, mysql exporter, QAN RDS, QAN postgres, RDS exporter)
+    hence (pmm-agent, Node exporter, postgres exporter, mysql exporter, QAN RDS,
+    QAN postgres, RDS exporter, QAN PostgreSQL PgStatements Agent,
+    QAN PostgreSQL PgStatements Agent, QAN MySQL Slowlog Agent)
      */
-    assert.ok((otherDetails <= 7 && otherDetails >= 4), 'Total Agents running on PMM-Server Instance can not be greater then 7');
+    assert.ok((otherDetails <= 10 && otherDetails >= 4), 'Total Agents running on PMM-Server Instance can not be greater then 10');
```

Comment:
why do we do this assertion?

Reply:
This assertion is part of the PMM-T345. Because of new added RDS PostgreSQL the total agents count increased to 10. It's because of 3 agents that are connected with RDS PostgreSQL. The test PMM-T345 has failed so I fixed it. So this assertion is not connected with the task.

Reply:
Ok, but why do we even need to do this? @puneet0191 It correct me if I'm wrong but when we will add a new instance for tests we will need to update this assertion, and at some point, it will be like `>4 && <20` or something like that.

Reply:
No it only happens when we are adding something on PMM-server via remote option, we can add many instances on client nodes this won't change the value here